pkgs/build-support/build-fhs-userenv/chrootenv/default.nix

```diff
@@ -2,7 +2,8 @@
 
 stdenv.mkDerivation {
   name = "chrootenv";
-  src = ./.;
+  src = builtins.filterSource
+      (path: _: builtins.baseNameOf path != "default.nix") ./.;
```

Comment:
there is another alternative, and is moving [chrootenv.c](https://github.com/NixOS/nixpkgs/blob/88933c7a0f549bfc9eefe64279c193e43a395eeb/pkgs/build-support/build-fhs-userenv/chrootenv/chrootenv.c) and [meson.build](https://github.com/NixOS/nixpkgs/blob/88933c7a0f549bfc9eefe64279c193e43a395eeb/pkgs/build-support/build-fhs-userenv/chrootenv/meson.build) to a folder, and then doing `src = ./folder`

Reply:
that's probably cleaner + simpler.. updated